exercises/sum.cpp

```diff
@@ -6,3 +6,27 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include<iostream>
+
+float sum(float num1, float num2){
+
+  return num1 + num2;
+
+}
+
+int main(){
+ float  num1, num2;
+ std::cout << "Insert the first number: ";
+ std::cin >> num1 ;
+
+ std::cout << "Insert the first number: ";
+ std::cin >> num2 ;
+ float ris = sum(num1, num2);
+ std::cout << "Sum: " <<ris  <<std::endl; 
+  
+
```

Comment:
manca return 0, inoltre potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```